### PR TITLE
Tenant level roles

### PR DESCRIPTION
**Pull request by aviadl**
+ tests
+ readme 
related to https://github.com/descope/etc/issues/2563

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -906,19 +906,21 @@ You can create, update, delete or load roles:
 
 ```typescript
 // You can optionally set a description and associated permission for a roles.
+// setting tenant id - will create this role for this specific tenant scope, leave empty to create a role taht will be allowed on all tenants
```
Comment (asafshen) [inline README.md:909]: small typo, also some small recommendation 

```suggestion
// Set the tenant ID to create a role scoped to a specific tenant. Leave empty for a global role applicable across all tenants.
```

**Review by asafshen — APPROVED** [review]
nice

small recommendation for comment

File: README.md (hunk 1 above)
Comment (itaihanski) [inline README.md:909]:
```suggestion
// The optional `tenantId` will scope this role for a specific tenant. If left empty, the role will be available to all tenants.
```

Comment (aviadl) [reply README.md]: done

Comment (aviadl) [reply README.md]: Took the other suggestion

**Pull request merged**